src/app/views/form/form.component.html

```diff
@@ -79,7 +95,12 @@
       {{ charsRemaining$ | async }}/{{ maxCharacters }} characters left.
     </div>
     <mat-form-field appearance="fill">
-      <textarea matInput formControlName="comment" ngModel maxlength="150"></textarea>
+      <textarea
+        matInput
+        formControlName="comment"
+        ngModel
+        maxlength="150"
```

Comment:
This should be moved to the formControl

Reply:
Could you specify more?

Reply:
Angular has MaxLengthValidator. If you choose to use reactive forms, you should not use template driven validations. Stick to one solution.